example_configs/tomcat.json

```diff
@@ -0,0 +1,42 @@
+{
+  "lowercaseOutputName": true,
+  "lowercaseOutputLabelNames": true,
+  "rules": [
+    {
+      "pattern": "Catalina<type=GlobalRequestProcessor, name=\"(\w+-\w+)-(\d+)\"><>(\w+):",
+      "name": "tomcat_$3_total",
+      "labels": {
+        "protocol":"$1",
+        "port":"$2" },
+      "help": "tomcat global $3",
```

Comment:
Ah, I think @brian-brazil didn't mean to also lower-case all help strings, only metric names (anything in the "name" field).